src/Impl/RenderTarget/RenderWindow/RenderWindow.h

```diff
@@ -0,0 +1,43 @@
+#ifndef _RENDER_WINDOW_H_
+#define _RENDER_WINDOW_H_
+
+#include <SFML/Window.hpp>
+#include <SFML/Graphics.hpp>
+
+#include "Plug/Graphics.h"
+
+class RenderWindow : public plug::RenderTarget
```

Comment:
I think this should be named something like `SfmlRenderTarget` to avoid confusion with SDL2 implementation